data-prepper-plugins/opensearch-source/src/main/java/org/opensearch/dataprepper/plugins/source/opensearch/model/PITResponse.java

```diff
@@ -0,0 +1,24 @@
+/*
+ * Copyright OpenSearch Contributors
+ * SPDX-License-Identifier: Apache-2.0
+ */
+
+package org.opensearch.dataprepper.plugins.source.opensearch.model;
+
+import org.opensearch.client.json.JsonpDeserializer;
+
+public class PITResponse  {
+
+    public static JsonpDeserializer<PITResponse> PARSER;
```

Comment:
Do we need this? And if so, how can we better hide it?

Using public variable fields is often not recommended.